en/net/advanced-operations/working-with-pages/move-pages/_index.md

```diff
@@ -122,20 +133,31 @@ srcDocument.Save(dstFileName);
 The following code snippet shows you how to move a bunch of pages from one PDF document to another.
 
 ```csharp
-var srcFileName = "<enter file name>";
-var dstFileName = "<enter file name>";
-var srcDocument = new Document(srcFileName);
-var dstDocument = new Document();
-var pages = new []{ 1, 3 };
-foreach (var pageIndex in pages)
+private static void MovingBunchOfPagesFromOnePdfDocumentToAnother()
 {
-    var page = srcDocument.Pages[pageIndex];
-    dstDocument.Pages.Add(page);
-}                       
-// Save output files
-dstDocument.Save(dstFileName);
-srcDocument.Pages.Delete(pages);
-srcDocument.Save(srcFileName);
+    // The path to the documents directory
+    string dataDir = RunExamples.GetDataDir_AsposePdf();
+    // Document path
+    string srcFileName = dataDir + "input.pdf";
+    string destFileName = dataDir + "moved_out.pdf";
+    //Open documents
+    using (var srcDocument = new Aspose.Pdf.Document(srcFileName))
+    {
+        using (var dstDocument = new Aspose.Pdf.Document())
+        {
+            var pages = new[] { 1, 3 };
+            foreach (int pageIndex in pages)
```

Comment:
According to early poll results, using type instead of 'var' is acceptable in lines where the type is not evident. I will leave this issue unresolved for now.

Reply:
From my point of view, it is quite obvious here that the elements of the array are integers.